### PR TITLE
[NTUSER] Error of co_UserCreateWindowEx should display class name

### PR DESCRIPTION
**Pull request by katahiromz**
## Purpose

The error messsage of `co_UserCreateWindowEx` function was unfriendly to the debugger.

JIRA issue: N/A

## Proposed changes

- Show the window class name by using `"%wZ"` in `co_UserCreateWindowEx` function.

### CODE REVIEW

File: win32ss/user/ntuser/window.c [hunk 1]
```diff
@@ -2394,7 +2394,7 @@ co_UserCreateWindowEx(CREATESTRUCTW* Cs,
 
    if (Window->spwndNext || Window->spwndPrev)
    {
-      ERR("Window 0x%p has been linked too early!\n", Window);
+      ERR("Window 0x%p has been linked too early! (%wZ)\n", Window, ClassName);
```
Comment (HBelusca) [inline win32ss/user/ntuser/window.c:2397]: Not sure it is meaningful to show the class name here for that error.

File: win32ss/user/ntuser/window.c [hunk 2]
```diff
@@ -2293,7 +2293,7 @@ co_UserCreateWindowEx(CREATESTRUCTW* Cs,
                             dwVer );
    if(!Window)
    {
-       ERR("IntCreateWindow failed!\n");
+       ERR("IntCreateWindow (%wZ) failed!\n", ClassName);
```
Comment (HBelusca) [inline win32ss/user/ntuser/window.c:2296]:
```suggestion
       ERR("IntCreateWindow(%wZ) failed!\n", ClassName);
```

File: win32ss/user/ntuser/window.c [hunk 3]
```diff
@@ -2419,7 +2419,7 @@ co_UserCreateWindowEx(CREATESTRUCTW* Cs,
    Result = co_IntSendMessage(UserHMGetHandle(Window), WM_NCCREATE, 0, (LPARAM) Cs);
    if (!Result)
    {
-      ERR("co_UserCreateWindowEx(): NCCREATE message failed\n");
+      ERR("co_UserCreateWindowEx (%wZ): NCCREATE message failed\n", ClassName);
```
Comment (HBelusca) [inline win32ss/user/ntuser/window.c:2422]:
```suggestion
      ERR("co_UserCreateWindowEx(%wZ): NCCREATE message failed\n", ClassName);
```

File: win32ss/user/ntuser/window.c [hunk 4]
```diff
@@ -2479,7 +2479,7 @@ co_UserCreateWindowEx(CREATESTRUCTW* Cs,
    Result = co_IntSendMessage(UserHMGetHandle(Window), WM_CREATE, 0, (LPARAM) Cs);
    if (Result == (LRESULT)-1)
    {
-      ERR("co_UserCreateWindowEx(): WM_CREATE message failed\n");
+      ERR("co_UserCreateWindowEx (%wZ): WM_CREATE message failed\n", ClassName);
```
Comment (HBelusca) [inline win32ss/user/ntuser/window.c:2482]:
```suggestion
      ERR("co_UserCreateWindowEx(%wZ): WM_CREATE message failed\n", ClassName);
```

File: win32ss/user/ntuser/window.c (hunk 1 above)
Comment (katahiromz) [reply win32ss/user/ntuser/window.c]: Reverted.

File: win32ss/user/ntuser/window.c [hunk 5]
```diff
@@ -2194,7 +2194,7 @@ co_UserCreateWindowEx(CREATESTRUCTW* Cs,
    pti = GetW32ThreadInfo();
    if (pti == NULL || pti->rpdesk == NULL)
    {
-      ERR("Thread is not attached to a desktop! Cannot create window!\n");
+      ERR("Thread is not attached to a desktop! Cannot create window (%wZ)!\n", ClassName);
```
Comment (JoachimHenze) [inline win32ss/user/ntuser/window.c:2197]:
```suggestion
      ERR("Thread is not attached to a desktop! Cannot create window (%wZ)\n", ClassName);
```

File: win32ss/user/ntuser/window.c [hunk 6]
```diff
@@ -2229,7 +2229,7 @@ co_UserCreateWindowEx(CREATESTRUCTW* Cs,
     }
     else if ((Cs->style & (WS_CHILD|WS_POPUP)) == WS_CHILD)
     {
-         ERR("Cannot create a child window without a parent!\n");
+         ERR("Cannot create a child window (%wZ) without a parent!\n", ClassName);
```
Comment (JoachimHenze) [inline win32ss/user/ntuser/window.c:2232]:
```suggestion
         ERR("Cannot create a child window (%wZ) without a parent\n", ClassName);
```

File: win32ss/user/ntuser/window.c [hunk 7]
```diff
@@ -2293,7 +2293,7 @@ co_UserCreateWindowEx(CREATESTRUCTW* Cs,
                             dwVer );
    if(!Window)
    {
-       ERR("IntCreateWindow failed!\n");
+       ERR("IntCreateWindow(%wZ) failed!\n", ClassName);
```
Comment (JoachimHenze) [inline win32ss/user/ntuser/window.c:2296]:
```suggestion
       ERR("IntCreateWindow(%wZ) failed\n", ClassName);
```

**Pull request merged**